awx/main/models/workflow.py

```diff
@@ -885,3 +892,12 @@ def workflow_job_template(self):
     @property
     def workflow_job(self):
         return self.unified_job_node.workflow_job
+
+    def notification_data(self):
+        result = super(WorkflowApproval, self).notification_data()
+        result.update(
+            dict(
+                extra_vars=self.workflow_job.extra_vars,
```

Comment:
wait a minute, you used `self.display_extra_vars()` for workflow notification data, but then `self.workflow_job.extra_vars` for the approval node notification data. Shouldn't this be `self.workflow_job.display_extra_data()`?

Reply:
Good catch, we want display_ here since stuff could bleed through the workflow.